tooling/Slack-channels.md

```diff
@@ -0,0 +1,17 @@
+##Slack
+
+##Join slack: broadinstitute.slack.com
+
+Slack is a major communication method for the team for questions and discussion. Saturn-team is the primary team channel.
+
+Recommended channels: 
+
+* Saturn-team
```

Comment:
also just to be nit-picky I would recommend making 'Saturn-team' and 'Saturn' lowercase as that's slack's convention and the other channels in this list are lowercased

Reply:
Thanks for this, that's a good call! I'll change the formatting and add some descriptions for the channels.